Marlin/Marlin_main.cpp

```diff
@@ -4064,7 +4064,7 @@ inline void gcode_M31() {
   lcd_setstatus(buffer);
 
   SERIAL_ECHO_START;
-  SERIAL_ECHOPGM(MSG_PRINT_TIME " ");
+  SERIAL_ECHOPGM("Print time: ");
```

Comment:
So `MSG_PRINT_TIME` isn't used anymore, right? Should be removed from language files ASAP (#4395)